### PR TITLE
Example readme

### PR DESCRIPTION
**Pull request by iclementine**
add README for transformer_tts, waveflow and wavenet

### CODE REVIEW

File: examples/transformer_tts/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+```bash
+python train.py --data=ljspeech_wavenet/ --output=runs/test --device="gpu" --nprocs=1
+```
+
+If you want distributed training, set a larger `--nprocs` (e.g. 4). Note that distributed training with cpu is not supported yet.
+
+## Synthesize
+
+Synthesize waveform. We assume the `--input` is text file, a sentence per line, and `--output` directory a directory to save the synthesized mel spectrogram(log magnitude) in `.npy` format. The mel spectrogram can be used with `Waveflow` to generate waveforms.
```
Comment (lfchener) [inline examples/transformer_tts/README.md]: directory a directory ？

File: examples/waveflow/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Extract the dataset.
+
+```bash
+tar xjvf LJSpeech-1.1.tar.bz2
+```
+
+### Preprocess the dataset. 
+
+Assume the path to save the preprocessed dataset is `ljspeech_wavenet`. Run the command below to preprocess the dataset.
```
Comment (lfchener) [inline examples/waveflow/README.md]: ljspeech_wavenet -> ljspeech_waveflow

File: examples/waveflow/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+```bash
+python preprocess.py --input=LJSpeech-1.1/  --output=ljspeech_wavenet
+```
+
+## Train the model
+
+The training script requires 4 command line arguments.
+`--data` is the path of the training dataset, `--output` is the path of the output direcctory (we recommend to use a subdirectory in `runs` to manage different experiments.)
```
Comment (lfchener) [inline examples/waveflow/README.md]: output directory

File: examples/waveflow/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+`--device` should be "cpu" or "gpu", `--nprocs` is the number of processes to train the model in parallel.
+
+```bash
+python train.py --data=ljspeech_wavenet/ --output=runs/test --device="gpu" --nprocs=1
```
Comment (lfchener) [inline examples/waveflow/README.md]: ljspeech_waveflow

File: examples/waveflow/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+If you want distributed training, set a larger `--nprocs` (e.g. 4). Note that distributed training with cpu is not supported yet.
+
+## Synthesize
+
+Synthesize waveform. We assume the `--input` is a directory containing several mel spectrogram(log magnitude) in `.npy` format. The output would be saved in `--output` directory, containing several `.wav` files with the same name as the mel spectrogram does.
+
+`--checkpoint_path` should be the path of the parameter file (`.pdparams`) to load. Note that the extention name `.pdparmas` is not included here.
+
+`--device` specifiies to device to run synthesis. Due to the autoregressiveness of wavenet, using cpu may be faster.
```
Comment (lfchener) [inline examples/waveflow/README.md]: waveflow？

File: examples/transformer_tts/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+### Extract the dataset.
+
+```bash
+tar xjvf LJSpeech-1.1.tar.bz2
+```
+
+### Preprocess the dataset. 
+
+Assume the path to save the preprocessed dataset is `ljspeech_wavenet`. Run the command below to preprocess the dataset.
```
Comment (lfchener) [inline examples/transformer_tts/README.md]: transformertts？

File: examples/transformer_tts/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+```bash
+python preprocess.py --input=LJSpeech-1.1/  --output=ljspeech_wavenet
```
Comment (lfchener) [inline examples/transformer_tts/README.md]: transformertts?

File: examples/transformer_tts/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+If you want distributed training, set a larger `--nprocs` (e.g. 4). Note that distributed training with cpu is not supported yet.
+
+## Synthesize
+
+Synthesize waveform. We assume the `--input` is text file, a sentence per line, and `--output` directory a directory to save the synthesized mel spectrogram(log magnitude) in `.npy` format. The mel spectrogram can be used with `Waveflow` to generate waveforms.
+
+`--checkpoint_path` should be the path of the parameter file (`.pdparams`) to load. Note that the extention name `.pdparmas` is not included here.
+
+`--device` specifiies to device to run synthesis. Due to the autoregressiveness of wavenet, using cpu may be faster.
```
Comment (lfchener) [inline examples/transformer_tts/README.md]: wavenet？

File: examples/wavenet/README.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+Assume the path to save the preprocessed dataset is `ljspeech_wavenet`. Run the command below to preprocess the dataset.
+
+```bash
+python preprocess.py --input=LJSpeech-1.1/  --output=ljspeech_wavenet
+```
+
+## Train the model
+
+The training script requires 4 command line arguments.
+`--data` is the path of the training dataset, `--output` is the path of the output direcctory (we recommend to use a subdirectory in `runs` to manage different experiments.)
```
Comment (lfchener) [inline examples/wavenet/README.md]: directory

File: examples/wavenet/README.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+If you want distributed training, set a larger `--nprocs` (e.g. 4). Note that distributed training with cpu is not supported yet.
+
+## Synthesize
+
+Synthesize waveform. We assume the `--input` is a directory containing several mel spectrogram(normalized into range[0, 1)) in `.npy` format. The output would be saved in `--output` directory, containing several `.wav` files with the same name as the mel spectrogram does.
+
+`--checkpoint_path` should be the path of the parameter file (`.pdparams`) to load. Note that the extention name `.pdparmas` is not included here.
+
+`--device` specifiies to device to run synthesis. Due to the autoregressiveness of wavenet, using cpu may be faster.
```
Comment (lfchener) [inline examples/wavenet/README.md]: specifiies？

File: examples/transformer_tts/README.md (hunk 1 above)
Comment (iclementine) [reply examples/transformer_tts/README.md]: is a..

File: examples/waveflow/README.md (hunk 2 above)
Comment (iclementine) [reply examples/waveflow/README.md]: done

File: examples/waveflow/README.md (hunk 3 above)
Comment (iclementine) [reply examples/waveflow/README.md]: done

File: examples/waveflow/README.md (hunk 4 above)
Comment (iclementine) [reply examples/waveflow/README.md]: done

File: examples/waveflow/README.md (hunk 5 above)
Comment (iclementine) [reply examples/waveflow/README.md]: I've removed that.

File: examples/transformer_tts/README.md (hunk 6 above)
Comment (iclementine) [reply examples/transformer_tts/README.md]: done

File: examples/transformer_tts/README.md (hunk 7 above)
Comment (iclementine) [reply examples/transformer_tts/README.md]: done

File: examples/transformer_tts/README.md (hunk 8 above)
Comment (iclementine) [reply examples/transformer_tts/README.md]: done.

File: examples/wavenet/README.md (hunk 9 above)
Comment (iclementine) [reply examples/wavenet/README.md]: done

File: examples/wavenet/README.md (hunk 10 above)
Comment (iclementine) [reply examples/wavenet/README.md]: specifies, done.

**Review by lfchener — APPROVED** [review]
LGTM

**Pull request merged**